src/main/java/frc/robot/RobotContainer.java

```diff
@@ -32,6 +32,12 @@ public class RobotContainer {
   private final CommandXboxController m_driverController = new CommandXboxController(
       OperatorConstants.kDriverControllerPort);
 
+  public Command driveFieldOrientedAnglularVelocity = m_swerveSubsystem.driveCommand(
```

Comment:
No need to make it `public`, since you should only be accessing it from within this same class